CHANGES.md

```diff
@@ -118,7 +118,7 @@ past jobs to be omitted in the UI.
 
 [#4860](https://github.com/cylc/cylc-flow/pull/4860) - Workflow validation
 now fails if
-[owner setting](https://cylc.github.io/cylc-doc/latest/html/reference/config/workflow.html#flow.cylc[runtime][%3Cnamespace%3E][remote]owner)
+[owner setting](https://cylc.github.io/cylc-doc/8.0.0/html/reference/config/workflow.html#flow.cylc[runtime][%3Cnamespace%3E][remote]owner)
```

Comment:
Might be better to go with `latest` for these entries.

Both links to `8.0.0` and `latest` will get broken in the long run, but, I think the `8.0.0` link is more likely to get broken sooner.